notify/notify.go

```diff
@@ -600,7 +600,11 @@ func (r RetryStage) Exec(ctx context.Context, l log.Logger, alerts ...*types.Ale
 		case <-tick.C:
 			if retry, err := r.integration.Notify(ctx, alerts...); err != nil {
 				numFailedNotifications.WithLabelValues(r.integration.name).Inc()
-				level.Debug(l).Log("msg", "Notify attempt failed", "attempt", i, "integration", r.integration.name, "err", err)
+				var alertnames []string
```

Comment:
one final nit .. could you change this to
```go
alertnames := make([]string, 0, len(alerts))
```

makes sure we only allocate memory once.

Reply:
sure! :)